en/documentation/templates/render.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: home
+title: Render
+sidebar: templates
+lang: en
+subnav: templates_render
+---
+
+# Render
+
+The render function takes a mandatory argument ```action```, that is the method of the action. It supports ```a:b:c``` notation. 
+Example:
+
+```smarty
+{render action="Module:Foo:bar"}
+```
+
+Will executes ```Module\Controller\FooController::barAction``` and print the result.
+
+__You can specify the request method__:
+
+```smarty
+{render action="Module:Foo:bar" method="PUT"}
+```
+
+__You can specify the query parameters__:
+
+```smarty
+{render action="Module:Foo:bar" query="foo=bar&baz=thelia"}
+{render action="Module:Foo:bar" query=$anArray}
+```
+
+__Same for request__ _(if the method isn't specified, it will automaticly set it to POST)_:
+
+```smarty
+{render action="Module:Foo:bar" request="foo=bar&baz=thelia"}
+{render action="Module:Foo:bar" request=$anArray}
+```
+

```

Comment:
This is the query string, it has to be the same in a URL.
Both ```request="foo=bar with space"```` or ```request="foo=bar+with+space"``` works.
The request parameter emulates a request body (a form, for example).

But it can be a good thing to know that if you want a literal "&" in your variable, you must escape it (```%26```)